code/modules/vending/clothesmate.dm

```diff
@@ -209,7 +209,7 @@
 		/obj/item/clothing/under/costume/swagoutfit = 1,
 		/obj/item/clothing/shoes/swagshoes = 1,
 		/obj/item/instrument/piano_synth/headphones/spacepods = 1,
-		/obj/item/clothing/head/helmet/toggleable/pinwheel = 1, //BUBBER EDIT: Pinwheel hat
+		/obj/item/clothing/head/helmet/toggleable/pinwheel = 1, //BUBBER EDIT: Pinwheel hat // BUBBER TODO: Modularize
```

Comment:
Skyrat wrote its own modularization system for the vendors btw

Reply:
I know, I am gonna include it right after this is finished, individually